bootstrap/script

```diff
@@ -69,5 +90,9 @@ say "Setting up project configurations... 🚀🔥\n"
 copyCodeStyleConfig
 copyKotlinCodeInsights
 
+say "Setting up plugins... 🚀🔥\n"
+copyAndUnzipPlugin plugin_adb_idea_1.5.4.zip
+copyAndUnzipPlugin plugin_android_drawable_importer_0.6.zip
```

Comment:
How often do you think we should update those plugins versions? 

In theory, the IDE would update those for us. However, if too much time goes by, we will start having major updates.

Do you think it could be an issue down the line? 🤔

Reply:
I don't think there's any need for updating these plugins. Unless it gets to a point where it's unusable with a new version of Android Studio ☠️ 

Will confirm later if it's still working with Android Studio version 4.0.0